appveyor.yml

```diff
@@ -20,11 +20,28 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
+      CONDA_PY: 27
       CONDA_INSTALL_LOCN: C:\\Miniconda
 
     - TARGET_ARCH: x64
+      CONDA_PY: 27
       CONDA_INSTALL_LOCN: C:\\Miniconda-x64
 
+    - TARGET_ARCH: x86
+      CONDA_PY: 34
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 34
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x86
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
```

Comment:
> You mean merge with the master branch of my own fork right? The rest will automatically take care of itself?

That is correct.

> Sorry I sound like a newb here, but this stuff is quite daunting; you're pulling all kinds of Github magic here I've never seen before

Don't worry. I get lost all the time and I am here since day 0 :smile: